### PR TITLE
Combined language file update 3

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Combined PR for the removal of unused and duplicated strings raised by horus68 in #3550 

## Motivation and Context

- Issue reference:  #3618 
- Issue reference:  #3602  

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/language/en_us.lang.php [hunk 1]
```diff
@@ -1595,7 +1595,6 @@
     'LBL_DIRECT_REPORTS' => 'Direct Reports',
     'LBL_DONE_BUTTON_LABEL' => 'Done',
     'LBL_DONE_BUTTON_TITLE' => 'Done',
-    'LBL_DST_NEEDS_FIXIN' => 'The application requires a Daylight Saving Time fix to be applied.  Please go to the <a href="index.php?module=Administration&action=DstFix">Repair</a> link in the Admin console and apply the Daylight Saving Time fix.',
     'LBL_DST_NEEDS_FIXIN' => 'The application requires a Daylight Saving Time fix to be applied. Please go to the <a href="index.php?module=Administration&action=DstFix">Repair</a> link in the Admin console and apply the Daylight Saving Time fix.',
     'LBL_EDIT_AS_NEW_BUTTON_LABEL' => 'Edit As New',
     'LBL_EDIT_AS_NEW_BUTTON_TITLE' => 'Edit As New',
```
Comment (horus68) [inline include/language/en_us.lang.php]: Also duplicated:
`'LBL_EDIT_AS_NEW_BUTTON_TITLE' => 'Edit As New',`

**Comment by samus-aran** [comment]
Hi @horus68 - These should be the reviews pulled in from the other PR. Possible to have a quick check and we'll merge it in asap?

**Comment by horus68** [comment]
@samus-aran Looks good from me.

**Comment by horus68** [comment]
@daniel-samson > We need to double check for labels which are built dynamically.

That could be the problem holding all those unused strings issues linked in the 
https://github.com/salesagility/SuiteCRM/issues/2444

**Comment by samus-aran** [comment]
@horus68 We are looking at the dynamically generated languages and how to best to catch them going forward, :+1:

**Pull request merged**